android/app/lombok.config

```diff
@@ -0,0 +1,2 @@
+lombok.anyConstructor.suppressConstructorProperties = true
+lombok.addGeneratedAnnotation = false
```

Comment:
@iamareebjamal Thanks :smile:  i tried removing JSON ignore properties then tested it in both case with `noIsPrefix` true and false. And got the point